### DISCUSSION TITLE
Custom Themes

### DISCUSSION THREAD

**Discussion by loctran016** [vuepress-theme-hope/vuepress-theme-hope#4582] [Q&A]
I want to add a custom theme (beside darkmode) that use different handwritten fonts and text colors, but not sure how to implement this feature with theme toggler components from the vuepress-theme-hope?

**Comment by Mister-Hope** [comment]
This sounds like you need to heavily override theme components, so you'd better be ready for a huge customization.

**Reply by loctran016** [reply]
Hmm, just fonts and bg Image, what I find struggling is children under .theme-hope-content doesn't have classes (it break with markmap but i use pretty dirty workaround 😶‍🌫️)

**Reply by Mister-Hope** [reply]
Why it's breaking markmap?

**Reply by loctran016** [reply]
Hmm cuz I use line height for .theme-hope-content p breaking markmap
